plugins/org.jkiss.dbeaver.registry/src/org/jkiss/dbeaver/registry/NativeClientDescriptor.java

```diff
@@ -51,14 +55,28 @@ public String getLabel() {
         return label;
     }
 
+    //Clients can't work from AppData folder, if app is Windows store app we don't want to create any native clients there
+    //see #15361
+    private boolean isClientsPathVirtualizedByWindows(File clientFile) {
```

Comment:
Let's use `Path` from NIO API to avoid converting back and forth between them.

```suggestion
    private boolean isClientsPathVirtualizedByWindows(@NotNull Path clientFile) {
```